### DISCUSSION TITLE
How to `dvc get` for a specific remote?

### DISCUSSION THREAD

**Discussion by vijay120** [treeverse/dvc#6447] [Help]
Currently, I have two remotes in repoA: `staging` and `production`. I also have a repoB service which consumes models from repoA. repoB needs to consume the staging model in repoA for it's staging env and production model in repoA for it's production env. I use `dvc get` to download the repoA model to repoB, but I cannot download the model based on the remote since it always downloads the default configured remote in repoA. 

Is there functionality where I can download models in another repo based on remote?

Thanks a lot and awesome tool!

**Comment by efiop** [comment]
Hi @vijay120 

Unfortunately this is not supported right now for `get/import/list`. We have a ticket for it though: https://github.com/iterative/dvc/issues/2466 No particular ETAs on that one, unfortunately :slightly_frowning_face: I suppose the remotes that you want to switch between are already defined in the target repo, right? If so, supporting `-r` for those cases should be fairly straightforward.
